README.md

```diff
@@ -302,12 +302,6 @@ It can be daunting to turn on strict deps checking or unused dependency mode che
 
 We recommend turning on strict_deps_mode first, as rule `A` might have an entry `B` in its `deps`, and `B` in turn depends on `C`. Meanwhile, the code of `A` only uses `C` but not `B`. Hence, the unused dependency checker, if on, will request that `B` be removed from `A`'s deps. But this will lead to a compile error as `A` can no longer depend on `C`. However, if strict dependency checking was on, then `A`'s deps is guaranteed to have `C` in it.
 
-### [Experimental] Migrating from deprecated configurations
-
-There are a few deprecated configuration methods which we will be removing in the near future.
```

Comment:
maybe we should leave a note somewhere to say that the java code can be controlled via strict_java_deps like java_library?

Reply:
Ok will add something